test/unit/src/extras/curves/NURBSCurve.tests.js

```diff
@@ -0,0 +1,70 @@
+/* global QUnit */
+
+import { NURBSCurve } from '../../../../../examples/jsm/curves/NURBSCurve.js';
+import { MathUtils } from '../../../../../src/math/MathUtils.js';
+import { Vector4 } from '../../../../../src/math/Vector4.js';
+
+export default QUnit.module( 'Extras', () => {
```

Comment:
Do you mind moving this unit test into the `test/unit/addons` folder? The `src` directory is intended for core modules only.

Reply:
Good suggestion, have done so